### DISCUSSION TITLE
v0.47.6 doesn't start

### DISCUSSION THREAD

**Discussion by hokutonanto** [TriliumNext/Trilium#2118] [Ideas]
Hi there,

I've updated Trilium to 0.47.6, when I open it I get this error message:
![image](https://user-images.githubusercontent.com/23059969/130339656-1da7d5cb-7bbf-4236-a337-61089194474e.png)

I'm on Windows 10

**Comment by TheNomad11** [comment]
Same on Linux Mint. I deinstalled it again and reinstalled 0.47.5 which works fine

**Comment by zadam** [comment]
https://github.com/zadam/trilium/issues/2117
